src/main/java/org/forgerock/openam/examples/SampleAuth.java

```diff
@@ -42,67 +42,72 @@
 import com.sun.identity.shared.datastruct.CollectionHelper;
 import com.sun.identity.shared.debug.Debug;
 
-
-
-public class SampleAuth extends AMLoginModule
-{
+/**
+ * SampleAuth authentication module example.
+ *
+ * If you want to create your own module based on this example, you will need to modify every "SampleAuth" occurrences
+ * in addition of refactoring this class name.
+ *
+ * Please refer to OpenAM documentation for further information.
+ *
+ * Feel free to have a look on the embedded authentication modules in OpenAM, as they implement the same API.
```

Comment:
Feel free to look at the code for authentication modules delivered with OpenAM, as they implement this same API.